src/ui/views/Approval/components/Actions/AssetOrder.tsx

```diff
@@ -149,37 +149,50 @@ const AssetOrder = ({
         </Col>
         <Col>
           <Row isTitle>{t('page.signTx.assetOrder.receiveAsset')}</Row>
-          <Row className="gap-y-6 flex flex-col overflow-hidden items-end">
-            {actionData.receiveTokenList.map((token) => (
-              <LogoWithText
-                className="overflow-hidden w-full"
-                key={token.id}
-                logo={token.logo_url}
-                text={
-                  <div className="overflow-hidden overflow-ellipsis flex">
-                    <Values.TokenAmount value={token.amount} />
-                    <span className="ml-2">
-                      <Values.TokenSymbol token={token} />
-                    </span>
-                  </div>
+          <Row>
```

Comment:
```suggestion
          <Row className="w-0">
```

Reply:
why?

Reply:
Avoid child elements that exceed the width of the parent element.
Or use `overflow-hidden`, but this will hide the security tag